desktop/webpack.config.development.js

```diff
@@ -22,6 +25,31 @@ config.output.publicPath = 'http://localhost:4000/dist/'
 // Uncomment below to figure out packaging bugs
 // config.bail = true
 
+config.plugins.push(new UnusedFilesWebpackPlugin({
+  pattern: '../shared/**/*.js',
+  globOptions: {
+    ignore: [
+      // Mobile stuff
+      '../**/*.native.js',
+      '../**/*.ios.js',
+      '../**/*.android.js',
+      // Flow stuff
+      '../shared/constants/folders.js',
```

Comment:
these things are only used in a flow context so they're invisible to webpack so we just ignore them